x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/geo_point_type.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import { NormalizedField, Field as FieldType } from '../../../../types';
+import { UseField, TextAreaField } from '../../../../shared_imports';
+import { IgnoreMalformedParameter, NullValueParameter } from '../../field_parameters';
+import { EditFieldSection, EditFieldFormRow, AdvancedSettingsWrapper } from '../edit_field';
+
+const getDefaultValueToggle = (param: string, field: FieldType) => {
+  switch (param) {
+    case 'null_value': {
+      return field.null_value !== undefined && field.null_value !== '';
+    }
+    default:
+      return false;
+  }
+};
+
+interface Props {
+  field: NormalizedField;
+}
+
+export const GeoPointType = ({ field }: Props) => {
```

Comment:
Same question here: don't you think it would be better to show some settings directly instead of forcing the user to click "Show advanced settings" ?

Reply:
I'd think I'd like to defer to the ES team and get their feedback. If it makes sense to pull some of them out of Advanced, I'm ok with that, but I don't know enough about this so I'd rather not arbitrarily choose some :)